modules/_labs/side-panel/react/lib/SidePanel.tsx

```diff
@@ -114,19 +115,13 @@ const SidePanel = ({
   onStateTransition,
   origin = 'left',
   variant = 'standard',
+  touched,
```

Comment:
I'm open to other names for this prop. It really means "the expanded state of the component has had intentional interaction.

Reply:
I'm okay with this. Add some JSDoc and I think that's good. I'm not sure there's a more succinct word to describe this variable.

Reply:
Done. Thanks!